rfc/seo.md

```diff
@@ -0,0 +1,72 @@
+# Gruntwork.io SEO RFC
+
+I read a lot of material in SEO, including going through a [guide to SEO](https://moz.com/beginners-guide-to-seo/).
+
+## Important take aways
+
+- We currently have some broken stuff in our website that should take priority over trying to improve SEO by creating more content. Ranked by (my perceived) importance:
+  - Searching for site:gruntwork.io/guides will show us that our content is duplicated between the www and non www versions of the website. This is diluting our ranking. We need to add a canonical link to our headers
+  - Our posts don't have meta descriptions
+  - Our internal links usually have the same anchor text (the title of the guide), but Google interprets that as not being natural and will penalize us for that
```

Comment:
Really? It used to be that you put into the link the _exact_ text you want to be ranked for... I wonder if this is some new rule?

Reply:
Yes and no! 
We do want the text we want to rank to be represented. But the question here is
```
<a href="...">How to deploy a production-grade Kubernetes cluster on AWS</a>
<a href="...">How to deploy a production-grade Kubernetes cluster on AWS</a>
<a href="...">How to deploy a production-grade Kubernetes cluster on AWS</a>
```
vs.
```
our <a href="...">guide on deploying dockerized containers on AWS</a>
you can <a href="...">learn how to configure a Kubernetes cluster</a>
we have a <a href="...">whole deployment walkthrough for EKS</a>
```

Quoted from the SEO guide I followed:
>Too many internal links using the same, keyword-stuffed anchor text can appear to search engines that you’re trying to manipulate a page’s ranking. It’s best to make anchor text natural rather than formulaic.

Reply:
Good to know, thx!

Reply:
(BTW, consider adding this example to the RFC)